website/siteConfig.js

```diff
@@ -180,7 +180,11 @@ const siteConfig = {
   highlight: {
     theme: 'atom-one-dark',
   },
-  scripts: ['https://buttons.github.io/buttons.js'],
+  scripts: [
+    'https://buttons.github.io/buttons.js',
+    'https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.0/clipboard.min.js',
```

Comment:
These two scripts have to be shifted into either the default `siteConfig` or into the HTML of the base body structure. Putting it here only affects Docusaurus' website, not its users.

Reply:
That makes sense. I kept these here and copied them to `examples/basics/siteConfig.js`, following buttons.github.io. Is that what you meant?